src/main/resources/assets/securitycraft/lang/fr_fr.json

```diff
@@ -0,0 +1,767 @@
+	"help.securitycraft.universal_key_changer.info": "Le changeur de code universel vous permet de changer le mot de passe des blocs protégés par mot de passe que vous possédez. Cliquez droit sur un bloc protégé par mot de passe pour ouvrir une interface graphique, puis saisissez le nouveau mot de passe souhaité. Créez-le avec une mallette pour en supprimer le code.",
+	"help.securitycraft.secret_sign_item.info": "Le Panneau à Secret fonctionne comme un panneau standard, à la différence que seul le propriétaire du panneau peut voir son texte. Assurez que personne ne puisse détruire le bloc sur lequel il est placé!",
+	"help.securitycraft.sentry.info": "La Sentinelle a trois modes qui peuvent être commutés en cliquant dessus avec le bouton droit. Inactif (il n’attaquera jamais), Camouflage (il n’attaquera que les joueurs ennemis à sa portée) et Agressif (il attaquera les mobs hostiles et les joueurs ennemis). Un clic droit avec un module de camouflage contenant un bloc plein et opaque (par exemple de la pierre, mais pas de verre ou un escalier) camouflera la Sentinelle elle-même. Le module peut être commuté directement ou extrait par un clic droit avec un Modificateur de Bloc universel. Pour retirer la sentinelle, faites un clic droit dessus.",
+	"help.securitycraft.crystal_quartz_item.info": "Le cristal de quartz est utilisé pour fabriquer des blocs de quartz de cristal.",
+
+	"gui.securitycraft:scManual.disabled": "Désactivé",
+	"gui.securitycraft:scManual.recipe.keycard_lv1": "Désactivé",
+	"gui.securitycraft:scManual.recipe.keycard_lv2": "Désactivé",
+	"gui.securitycraft:scManual.recipe.keycard_lv3": "Désactivé",
+	"gui.securitycraft:scManual.recipe.keycard_lv4": "Désactivé",
+	"gui.securitycraft:scManual.recipe.keycard_lv5": "Désactivé",
+	"gui.securitycraft:scManual.recipe.limited_use_keycard": "Désactivé",
+	"gui.securitycraft:scManual.intro.1": "Démarrer avec SecurityCraft",
+	"gui.securitycraft:scManual.intro.2": "par Geforce et bl4ckscor3",
+	"gui.securitycraft:scManual.ownableBloc": "Propriétaire: ce bloc résiste aux explosions et ne peut être cassé que par le joueur qui l’a déposé à l’aide du Déplaceur De bloc universel.",
+	"gui.securitycraft:scManual.passwordProtectedBloc": "Protégé par Mot de passe : ce bloc ne peut être utilisé que si un mot de passe défini par le propriétaire est correctement entré.",
+	"gui.securitycraft:scManual.viewActivatedBloc": "Affichage activé: ce bloc peut être activé en le survolant avec votre curseur lorsque vous êtes à portée.",
+	"gui.securitycraft:scManual.explosiveBloc": "Explosif: ce bloc peut être déclenché à distance, et peut être activé / désactivé, à l’aide de l’outil d’accès aux mines à distance.",
+	"gui.securitycraft:scManual.customizableBloc": "Personnalisable: vous pouvez utiliser le Modificateur de Bloc universel et les différents modules ajoutés par SecurityCraft pour personnaliser les fonctions de ce bloc.",
```

Comment:
You accidentally removed the `k` of `Block` at the end of each of these lines' language key.